### PR TITLE
Add CNAME records for queues.federalistapp.18f.gov

### PR DESCRIPTION
**Pull request by apburnes**
- ~This is a new public-facing site _(if so, please follow the additional instructions below)_~ (N/A)

This creates CNAMES to support queues.federalistapp.18f.gov which is an internal tool used by the federalist platform team.

### CODE REVIEW

File: terraform/18f.gov.tf [hunk 1]
```diff
@@ -1782,3 +1782,21 @@ resource "aws_route53_record" "d_18f_gov__acme_challenge_c2_18f_gov_cname" {
 output "d_18f_gov_ns" {
   value = aws_route53_zone.d_18f_gov_zone.name_servers
 }
+
+# queues.federalistapp.18f.gov - CNAME -----------------
+resource "aws_route53_record" "d_18f_gov_queues_federalistapp_18f_gov_cname" {
+  zone_id = "${aws_route53_zone.d_18f_gov_zone.zone_id}"
```
Comment (svenaas) [inline terraform/18f.gov.tf:1788]: @apburnes This is probably not a problem but I noticed that your syntax
```
  zone_id = "${aws_route53_zone.d_18f_gov_zone.zone_id}"
```
is different from that used for, say, the `c2.18f.gov` record a few blocks above:
```
  zone_id = aws_route53_zone.d_18f_gov_zone.zone_id
```
If either will work then I won't worry about it, but I wanted to note it.

**Review by svenaas — APPROVED** [review]
One comment (probably just for my own education), but otherwise this looks good to me.

File: terraform/18f.gov.tf (hunk 1 above)
Comment (apburnes) [reply terraform/18f.gov.tf]: Thanks! Updated for consistency.

**Review by svenaas — APPROVED** [review]
Well now I have no questions. Approved, and have a good weekend!

**Pull request merged**